addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -860,6 +860,28 @@ export function preserveCursor(document) {
     };
 }
 
+/**
+ * Check if the selection starts inside given selectors. This function can be
+ * used as the `isDisabled` property of a command of the PowerBox to disable
+ * a command in the given selectors.
+ * @returns {boolean} true selector is within one of the selectors
+ * (if the command should be filtered)
+ */
+export function isSelectionInSelectors(selectors) {
+    let anchor = document.getSelection().anchorNode;
+    if (anchor && anchor.nodeType && anchor.nodeType !== Node.ELEMENT_NODE) {
+        anchor = anchor.parentElement;
+    }
+    if (anchor) {
+        for (const idx in selectors) {
```

Comment:
I would use a multi-selector (one string with `, ` between selectors) instead of an array, so that we don't have to put a `for` loop here :)